spec.bs

```diff
@@ -1680,11 +1680,11 @@ partial interface Navigator {
   1. Let |instance| be [=this=]'s [=relevant global object=]'s [=Window/browsing context=]'s
      [=browsing context/fenced frame config instance=].
 
-  1. If |instance| is null, then throw a {{DOMException}}.
+  1. If |instance| is null, then [=exception/throw=] a {{DOMException}}.
```

Comment:
Now that I look at this, we have to specify the specific type of DOMException. Can you make this match the implementation?